libraries/heft-config-file/src/test/__snapshots__/ConfigurationFile.test.ts.snap

```diff
@@ -272,12 +272,12 @@ exports[`ConfigurationFile error cases Throws an error when a combined config fi
 "Resolved configuration object does not match schema: Error: JSON validation failed:
 <project root>/lib/test/errorCases/invalidCombinedFile/config1.json
 
-Error: #/
-       Data does not match any schemas from 'oneOf'
-  Error: #/
-         Additional properties not allowed: folderPaths
-  Error: #/
-         Additional properties not allowed: filePaths"
+Error: # (must NOT have additional properties)
```

Comment:
The parenthesis on these lines aren't particularly useful.

Reply:
yeah that is fair, ajv does not have 'detailed' error object like z-schema and present all relevant info in the error 'message'. so we can probably remove the parenthesis section completely